From 5ec89ed17be50b3ad815836d497f340568ce9a58 Mon Sep 17 00:00:00 2001
From: Chen Liangjun <b36089@freescale.com>
Date: Mon, 29 Oct 2012 15:38:26 +0800
Subject: [PATCH 2224/2508] ENGR00231773-7 DMA: add interface to trigger SDMA
 event pending bit

Add interface in header file to call sdma event pending trigger
function.

Signed-off-by: Chen Liangjun <b36089@freescale.com>
---
 arch/arm/plat-mxc/include/mach/dma.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/dma.h b/arch/arm/plat-mxc/include/mach/dma.h
index 3fccb3b..af2edb8 100644
--- a/arch/arm/plat-mxc/include/mach/dma.h
+++ b/arch/arm/plat-mxc/include/mach/dma.h
@@ -101,5 +101,6 @@ static inline int mxs_dma_is_apbx(struct dma_chan *chan)
 	return !strcmp(dev_name(chan->device->dev), "mxs-dma-apbx");
 }
 
+void sdma_set_event_pending(struct dma_chan *chan);
 
 #endif
-- 
1.7.7.4

